GOVERNANCE.md

```diff
@@ -1,132 +1,175 @@
 # OpenRefine Governance Model
 
 ## Summary / Overview
-OpenRefine is a free, open-source, powerful tool for working with messy data. OpenRefine has a plugin architecture and is distributed under the [new BSD license](http://opensource.org/licenses/BSD-3-Clause) allowing modification, distribution and name changes.
+OpenRefine is a free, open-source, powerful tool for working with messy data. OpenRefine has a [plugin architecture](https://openrefine.org/docs/technical-reference/writing-extensions) and is distributed under the [new BSD license](http://opensource.org/licenses/BSD-3-Clause) allowing modification, distribution and name changes.
 
 ## Roles and responsibilities
-OpenRefine development is based on user consensus and open discussion between users. Decision making must be done in a transparent, open fashion (i.e. using OpenRefine's discussion list and issue list). No decisions about the project’s direction, bug fixes or features may be done in private without community involvement and participation. Discussions must begin at the earliest possible point on a topic; the community’s participation is vital during the entire decision-making process.
+OpenRefine is a meritocratic, consensus-based community project based on the [Apache Decision Making process](https://community.apache.org/committers/decisionMaking.html).
 
-All project participants abide by the [Code of Conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md).
+Decision-making is done based on user consensus following open discussion (i.e., using OpenRefine's [discussion list](https://forum.openrefine.org/c/dev/8) and [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open)). No decisions about the project’s direction, bug fixes, or features may be made in private without community involvement and participation. Discussions must begin at the earliest possible point on a topic; the community’s participation is vital during the entire decision-making process.
 
-Anyone with an interest in the project can join the community, contribute to the project design, and participate in the decision making process. This document describes how that participation takes place.
+All project participants abide by the [Code of Conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md).
```

Comment:
Is the CoC up to date? Last time (which I know of) it was "enforced" by an AC member not listed in that doc...

Reply:
I think the members could be refreshed (for instance, as Jessica has left CS&S), but to me it still reflects the current state of the committee. Are you worried that the CoC committee could be just a sock puppet of the AC without actual independence and agency? If you are referring to me as a former AC member, last time I reached out to the CoC about a particular situation, they disagreed about my assessment of that situation and followed their own judgment, which I would take as a good sign of the committee's independence.

That being said, it's an open question to me how to keep a CoC committee active for such a small project because it only gets woken up once in a blue moon, making it hard to keep the practice and understandings of the tasks alive. Surely there is room for improvement, but that's something we should rather do in another PR I would say.

Reply:
Whether or not the CoC is mentioned here or not doesn't change its status I would hope. This is just a link for convenience.

Reply:
@Ainali the CoC contains quite a lot about governance and it's currently very badly aligned with this document, who are considered a "project leader"? can the CoC group enforce rights-changes against contributors? It's all governance, and with the ACs history of weaponizing it, I would ague it's rather important to get it right.